packages/composite-checkout/src/components/checkout-review-order.js

```diff
@@ -37,6 +38,7 @@ export default function CheckoutReviewOrder( { summary, className } ) {
 			<OrderReviewSection>
 				<OrderReviewTotal total={ total } />
 			</OrderReviewSection>
+			<TermsAndConditions />
```

Comment:
On most host pages, we'll be providing a custom review order step, which will be rendered in place of `CheckoutReviewOrder`, so `TermsAndConditions` will not be visible. Also, the contents of that component will be different on each host, so let's make this a slotted component instead.

Put `TermsAndConditions` into its own file and rename it to something that makes it clear that it's for WPCOM like `WPComTermsAndConditions`, and for now you can export that component as part of the public API.

Then in `demo/index.js`, pass it as a prop to `Checkout` like this: `<Checkout TermsAndConditions={ WPComTermsAndConditions } />`. In `Checkout` we can render it right before `CheckoutSubmitButton`. (See the `UpSell` prop for an example of this.) Since we probably want to hide and show this component along with the review step, put that logic in when it's rendered, like `{ stepNumber === 3 && TermsAndConditions && <TermsAndConditions /> }`.

Reply:
aah yes. That makes sense. I'll update it to work this way.

Reply:
@sirbrillig I got this all cleaned up and took a slightly different approach by passing the T&C component down the hierarchy so that it sits within the review component. Let me know what you think.